app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/recent/RecentFragment.java

```diff
@@ -13,11 +13,13 @@
 import com.fstyle.androidtrainning.data.remote.service.config.LastFmApi;
 import com.fstyle.androidtrainning.model.RecentTracks;
 import com.fstyle.androidtrainning.screen.BaseFragment;
+import com.fstyle.androidtrainning.screen.main.mainfragments.mysong.subfragment.listsong
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.screen.main.mainfragments.mysong.subfragment.listsong.OnFavoriteClick. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports